web/src/app/workplace/workbasket-selector/workbasket-selector.component.ts

```diff
@@ -61,4 +56,9 @@ export class SelectorComponent implements OnInit {
         tasks._embedded.tasks.forEach(e => this.tasks.push(e));
       });
   }
+
+  isDisabled(): boolean {
```

Comment:
this is not efficient, You are iterating over an array everytime something in the UI is refreshed.
You need to explain me what you want to do here.

Reply:
well i want the button to only be active, when the input field contains one of the actual workbasket names, since there is no reason to search for a non existing workbasket

Reply:
You are using, component "ng2-auto-complete": "0.12.0" for this... We have already imported ng-bootstrap components library, and you can use typeahead component which has already implemented onSelect(). You can use this onSelect() function to check if the workbasket is correct or not, simply using if(selected) then is correct else wrong ...